tests/integration/s2n_tls13_handshake_tests.py

```diff
@@ -60,30 +60,58 @@ def verify_hrr_random_data(server, client):
 
     return result
 
+def key_update_send_and_receive(client, server, key_update_request):
+    openssl_msg = "Message:" + str(uuid.uuid4())
+    s2n_msg = "Message:" + str(uuid.uuid4())
+
+    # Trigger Openssl to send a KeyUpdate message
+    client.stdin.write((key_update_request + "\n\n").encode("utf-8"))
+    client.stdin.flush()
+
+    # Confirm that the KeyUpdate was sent
+    time.sleep(0.01)
+    if not wait_for_output(client.stderr, 'KEYUPDATE', 100):
```

Comment:
Ahh I see. Ideally this will be deprecated soon anyway, so no blocking on function names.

Reply:
Yeah, I'm going to move the test over this week probably.